libpsautohint/autohintexe.c

```diff
@@ -402,7 +420,7 @@ main(int argc, char* argv[])
             }
 
             result = AutoColorString(bezdata, fontinfo, &output, &outputsize,
-                                     allowEdit, allowHintSub, roundCoords, debug);
+                                     allowEdit, allowHintSub, roundCoords);
```

Comment:
Removing debug argument will break the C api of libpsautohint. I guess next version should be 2.0

Reply:
We broke it many times before, and will probably do again. But since we don’t publish the C library independently I think it should be OK. One day we should stabilize it and make such a standalone release, but I want to rewrite the API before that.

Reply:
sgtm